db/seeds.rb

```diff
@@ -0,0 +1,26 @@
+User.create(email: 'bobbysags@gmail.com', name: 'Bob Saget', password: '123', age: 60,)
```

Comment:
This is fine, but I would use loops here to make this users.. I saw that you commented out the Faker users... Did you have trouble implementing that or you just chose to do it manually?